google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/RpcViewConstants.java

```diff
@@ -124,4 +128,22 @@
               BIGTABLE_APP_PROFILE_ID,
               BIGTABLE_OP,
               BIGTABLE_STATUS));
+
+  static final View BIGTABLE_GFE_LATENCY_VIEW =
+      View.create(
+          View.Name.create("cloud.google.com/java/bigtable/gfe_latency"),
+          "GFE t4t7 latency in msecs",
+          BIGTABLE_GFE_LATENCY,
+          AGGREGATION_WITH_MILLIS_HISTOGRAM,
+          ImmutableList.of(
+              BIGTABLE_INSTANCE_ID, BIGTABLE_PROJECT_ID, BIGTABLE_APP_PROFILE_ID, BIGTABLE_OP));
+
+  static final View BIGTABLE_GFE_HEADER_MISSING_COUNT_VIEW =
+      View.create(
+          View.Name.create("cloud.google.com/java/bigtable/gfe_header_missing_count"),
+          "Number of responses without the server-timing header",
```

Comment:
Can you add a note what this means? ie. that most likely the request never made it to the google's load balancers?